testui/support/testui_driver.py

```diff
@@ -457,9 +461,9 @@ def stop_recording_and_compare(
         """
         now = datetime.now()
         current_time = now.strftime("%Y-%m-%d%H%M%S")
+        log_dir = self.__configuration.screenshot_path
```

Comment:
This assignment is already being conditionally done on line 468. Should the condition be kept?

Reply:
Yes, missed that, it should be moved up, Ill change it